app/packages/addons-service/provider.ts

```diff
@@ -110,6 +110,23 @@ export default class Provider {
         return account;
     }
 
+    @task
+    @waitFor
+    async createConfiguredStorageAddon() {
+        if (!this.configuredStorageAddon) {
+            const configuredStorageAddon = this.store.createRecord('configured-storage-addon', {
+                externalUserId: this.currentUser.user?.id,
+                externalUserDisplayName: this.currentUser.user?.fullName,
```

Comment:
I think these are set by Gravy Valet after connecting to the external service and authenticating the user, then getting back the identifying information from that service. I don't believe we set it. This may require some Mirage gymnastics to replicate.